src/scripts/components/VariableMessage.jsx

```diff
@@ -18,24 +20,43 @@ import {
 } from '../constants';
 
 class VariableMessage extends React.Component {
+  currentDraggedSpanVariable = '';
+  isCategoryMobileViewVisible = false;
```

Comment:
Maintaining an instance property to determine whether to show the mobile view doesn't seem to allow us to react to changes in view width (I.E a user flipping their phone from portrait to landscape will break it). Seems to me this should be done with CSS/LESS breakpoints. Let me know what you think.